org.adempiere.base/src/org/compiere/model/ModelValidationEngine.java

```diff
@@ -573,9 +574,14 @@ public String fireDocValidate (PO po, int docTiming)
 		EventManager.getInstance().sendEvent(event);
 		@SuppressWarnings("unchecked")
 		List<String> errors = (List<String>) event.getProperty(IEventManager.EVENT_ERROR_MESSAGES);
-		if (errors != null && !errors.isEmpty())
-			return errors.get(0);
-
+		if (errors != null && !errors.isEmpty()) {
+			Collections.reverse(errors);
+			StringBuilder eventErrors = new StringBuilder("");
+			for (String error : errors) {
+				eventErrors.append(error).append("<br>");

```

Comment:
can you please test whether `\n` works for the UI too ? If that's working fine for UI as well, would prefer that over `<br>` since it will works fine for console or log file as well.

Reply:
@hengsin I tested it and it would have to be <br> the same for a better visualization, looking at the code I found other points that need the same treatment, I will adjust it too

Reply:
@hengsin ajusted